client/src/App.tsx

```diff
@@ -1,35 +1,24 @@
-import { useEffect, useState } from 'react';
-import reactLogo from './assets/react.svg';
-import viteLogo from '/vite.svg';
-import './App.css';
+// import React from 'react';
+import {
+  BrowserRouter as BrowserRouter,
+  Route,
+  Routes,
+} from 'react-router-dom';
+import { NotFound } from './pages/NotFound';
+import { Navbar } from './components/NavBar';
+import { SeriesList } from './pages/SeriesList';
+import './index.css';
+import { HomePage } from './pages/HomePage';
 
 export default function App() {
-  const [serverData, setServerData] = useState('');
-
-  useEffect(() => {
-    async function readServerData() {
-      const resp = await fetch('/api/hello');
-      const data = await resp.json();
-
-      console.log('Data from server:', data);
-
-      setServerData(data.message);
-    }
-
-    readServerData();
-  }, []);
-
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank" rel="noreferrer">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank" rel="noreferrer">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>{serverData}</h1>
-    </>
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
```

Comment:
Works, but if you want the `<Navbar />` to be visible on every page, that's where the `<Outlet />` comes into play. So your `<Navbar>` should be the parent `route` to all of these